OWNERS

```diff
@@ -20,4 +20,7 @@ approvers:
 - skabashnyuk
 - deboer-tim
 - ksprinkl
-- scoheb
\ No newline at end of file
+- scoheb
+- eisraeli
+- amisstear
```

Comment:
My username is `amisstea` :smile:

Reply:
Fixed.
Was it intentional that they dropped the `r` at the end ? :sweat_smile: